lib/manageiq/ssh/util.rb

```diff
@@ -1,6 +1,7 @@
 require 'net/ssh'
 require 'net/sftp'
 require 'tempfile'
+require 'active_support'
```

Comment:
I'm curious when/why this became a requirement, but https://github.com/rails/rails/issues/43889 suggests that it's necessary.

Reply:
That's the one